frontend/src/app/chatmenu/BlockUser.css

```diff
@@ -0,0 +1,79 @@
+.UserTile {
+    --image-size: 45px;
+    --margin-size: 5px;
+    font-size: 0;
+    display: block;
+    flex-direction: column;
+    justify-content: space-between;
+    align-content: center;
+}
+
+.UserRow .UserTile {
+    margin-top: var(--margin-size);
+    margin-right: var(--margin-size);
+    margin-left: 0;
+    margin-bottom: 0;
+}
+
+.UserRow .UserTile:last-child {
+    margin-right: 0;
+}
+
+.UserRowCentered .UserTile {
+    margin: var(--margin-size);
+}
+
+.UserTile .ProfileImage,
+.UserTile .DisplayName {
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.UserTile .ProfileImage {
+    width: var(--image-size);
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.UserTile .ProfileImage .Image {
+    width: 100%;
+}
+
+.UserTile .DisplayName {
+    font-size: var(--text-size);
+    padding: 0.5em;
+}
+
+.UserDisc {
+    vertical-align: middle;
+    display: inline-block;
+    width: 45px;
+    border-radius: 50%;
+    overflow: hidden;
+    font-size: 0;
+    margin: 5px;
+}
+
+.UserDisc .Image {
+    width: 100%;
+}
+
+button {
+    border: none;
+    border-radius: 25px;
+    padding: 5px;
+    display: inline-flex;
+}
```

Comment:
Good work on the CSS!

Optional, but I would also recommend adding a style rule for when the button is hovered and setting `cursor: pointer`, this changes the cursor to indicate that an element is clickable.